auth/pom.xml

```diff
@@ -12,33 +12,33 @@
   <parent>
     <groupId>edu.tamu.weaver</groupId>
     <artifactId>webservice-parent</artifactId>
-    <version>2.0.0</version>
+    <version>2.0.0-MESSAGING-MODULE-SNAPSHOT</version>
```

Comment:
It may be more appropriate to have it be 2.x-MESSAGING-MODULE-SNAPHOT. The eventual release will most likely be a minor version change. Will also need it to be deployed to our Maven Repository. @jcreel

Reply:
https://github.com/TAMULib/Weaver-Webservice-Core/commit/37a99d8ea7081b1ada78e7e6ae4c84d91db8228f